srcpkgs/OpenRCT2/template

```diff
@@ -49,5 +57,7 @@ post_extract() {
 	_srcdir="${XBPS_SRCDISTDIR}/${pkgname}-${version}"
 	mkdir -p data/object data/title
 	unzip -qd data/object "${_srcdir}/objects.zip"
```

Comment:
Please replace the two `unzip` commands with `bsdtar` ones, if possible, then remove `unzip` from `hostmakedepends`.